### PR TITLE
[16.0][MIG] sale_order_tag

### PR DESCRIPTION
**Pull request by Kev-Roche**
Migration of sale_order_tag from 14.0
Change menu name "Tags" into "Sale Order Tags" to avoid duplicate name with "Tags" from crm

### CODE REVIEW

**Comment by rousseldenis** [comment]
@Kev-Roche Thanks for this. Could you follow migration guide especially the full process part ? 

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

In fact, the setup/pre-commit stuff should go in a separate commit

**Comment by rousseldenis** [comment]
@Kev-Roche

**Comment by Kev-Roche** [comment]
@rousseldenis done, sorry for the delay.

File: sale_order_tag/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "author": "Patrick Wilson, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
     "category": "Sale",
-    "version": "14.0.1.0.1",
+    "version": "16.0.0.0.1",
```
Comment (rousseldenis) [inline sale_order_tag/__manifest__.py]: This should be `16.0.1.0.0`

**Review by rousseldenis — CHANGES_REQUESTED** [review]
@Kev-Roche Could you rearrange your changes in commits ?

In pre-commit should go ... pre-commit stuffs and in migration one, guess :smiley: 

Thanks

File: sale_order_tag/models/sale_order_tag.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    team_ids = fields.Many2many("crm.team", string="Sales Teams")
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one("res.company", string="Company")
+    sequence = fields.Integer(
+        default=lambda self: self.env["ir.sequence"].next_by_code("sale.order.tag")
+        or 0,
+        required=True,
+    )
+
+    _sql_constraints = [("name_uniq", "unique (name)", "Tag name already exists!")]
```
Comment (ferran-S73) [inline sale_order_tag/models/sale_order_tag.py]: Maybe it could be interesting to make tag names unique per company

Comment (Kev-Roche) [reply sale_order_tag/models/sale_order_tag.py]: Indeed, I added it, thank you.

**Review by ferran-S73 — APPROVED** [review]

**Review by hildickethan — APPROVED** [review]

**Comment by hildickethan** [comment]
pretty much everything in the pre-commit commit except the `setup` folder should be in the migration commit instead, with that I think we can merge

**Comment by ChrisOForgeFlow** [comment]
@Kev-Roche can you rebase PR please?

**Review by patel1008b — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**